### PR TITLE
Make pages more responsive

### PR DESCRIPTION
**Pull request by petey**
* Add purecss responsive grids css.
* Make Pipeline->Builds and Homepage scale to 1 unit grids when page is small
* Make header menu collapse into hamburger when page is too small
* Cut up header background image so that it scales properly in all the pieces as the page is resized.

Collapsed:
screen shot 2016-09-28 at 5 37 51 pm

Open:
screen shot 2016-09-28 at 5 38 02 pm

Addresses: https://github.com/screwdriver-cd/screwdriver/issues/214

### CODE REVIEW

File: app/components/pipeline-nav/styles.scss [hunk 1]
```diff
@@ -1,8 +1,11 @@
 & {
   padding-top: 5px;
-  background: url('/assets/main_bkgrd_home.png') transparent;
-  background-size: cover;
-  background-position: 0 -100px;
+  // background: url('/assets/main_bkgrd_home.png') transparent;
```
Comment (stjohnjohnson) [inline app/components/pipeline-nav/styles.scss]: Cleanup commented code

Comment (petey) [inline app/components/pipeline-nav/styles.scss]: fixed

**Comment by FenrirUnbound** [comment]
Build failed 😿

**Comment by minz1027** [comment]
The acceptance tests failed only because the div class for `<h2>Builds</h2>` changes. Except that others look great! 👍

**Pull request merged**